server/lib/images.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Returns true if image is a valid OC image uploaded in our S3 bucket.
+ * Allow any image on non-production environments
+ */
+export const isValidOCImage = (url: string): boolean => {
+  if (process.env.NODE_ENV !== 'production') {
+    return true;
+  } else {
+    return url.startsWith(`${process.env.AWS_S3_BUCKET}.s3-us-west-1.amazonaws.com`);
```

Comment:
@znarf do you think that's a safe way to check that uploaded image URLs are valid?

Reply:
It doesn't start with http ?

Reply:
We should not use `process.env` but use `config` to read configuration.